src/main/java/net/minestom/server/sound/SoundCategory.java

```diff
@@ -1,5 +1,9 @@
 package net.minestom.server.sound;
 
+/**
+ * @deprecated Use {@link net.kyori.adventure.sound.Sound.Source}
+ */
+@Deprecated
 public enum SoundCategory {
```

Comment:
Is there a reason why?
Also, does Adventure enum respects the order the MC protocol expects the values?

Reply:
I'll leave discussion of the deprecation of SoundCategory to the other thread, but this does raise the point that Adventure doesn't *guarantee* the ordering - so I'll some methods to the utils class.